com.aobuchow.themes.spectrum/css/default_overrides.css

```diff
@@ -273,37 +278,66 @@ Shell[style~='SWT.RADIO'][style~='SWT.CASCADE'][style~='SWT.SHADOW_ETCHED_IN'][s
 .MPart ScrolledComposite,
 .Mpart ScrolledComposite ProgressInfoItem,
 .MPart Form ScrolledPageBook,
-.MPart DependenciesComposite > SashForm > Section > * { /* Section > DependenciesComposite$... */
-    background-color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
-    color:#eeeeee;
+.MPart DependenciesComposite > SashForm > Section > * {
+  /* Section > DependenciesComposite$... */
+  background-color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
+  color: #eeeeee;
 }
 
 Shell,
-Composite, ScrolledComposite, ExpandableComposite, Canvas, TabFolder, CLabel, Label,
```

Comment:
Maybe a no-format should be used here? Would keep the structure similar to the platform dark theme's CSS, although it's more readable when each CSS element is on one line.

Reply:
@PyvesB WDYT? This is more a general readability question for devs, let me know if you have an opinion

Reply:
In my opinion, it's more readable with one item per line. Unless you've got a very good reason to do so, don't get into the habit of adding no-formats, otherwise you'll constantly start asking yourself whether you should add new ones here and there. We use Prettier for [Shields.io](https://github.com/badges/shields), I don't believe we have many of them lying around. :wink:

Reply:
I totally agree :) 1 item per line is more readable & using no-formats can quickly spiral out of control (+ adds a bit of maintenance overhead). Thanks for your input :)

Reply:
@PyvesB by the way, it might be nice to use some Shields for this repo :) Do you have any suggestions for shields to use? I’ve always wanted to use them for a repo 😎

Reply:
Some of the activity ones seem really great, especially the commit activity & Eclipse marketplace ones. 


https://shields.io/category/activity

Reply:
Yes, there are quite a few Eclipse Marketplace badges available, I recommend the downloads ones as the information is not directly available through the marketplace website. Favorites is a nice to have as well.

GitHub commit activity is indeed another good one. A lot of people add badges for number of contributors, stars, open issues, license, etc., but the information is also already available on the repo homepage, so they are less useful in my opinion.

Another idea: you could add a badge indicating the Prettier version used by the project for example (search for "package.json dependency version" in the Shields search bar).

Reply:
> Yes, there are quite a few Eclipse Marketplace badges available, I recommend the downloads ones as the information is not directly available through the marketplace website. Favorites is a nice to have as well.

Great idea +1 :)

> GitHub commit activity is indeed another good one. 

Right!

> A lot of people add badges for number of contributors, stars, open issues, license, etc., but the information is also already available on the repo homepage, so they are less useful in my opinion.

Agreed, no need for these.

> Another idea: you could add a badge indicating the Prettier version used by the project for example (search for "package.json dependency version" in the Shields search bar).

+1 :)

I'm going to make a new bug for this.